Source/WebCore/Modules/applepay/cocoa/PaymentAPIVersionCocoa.mm

```diff
@@ -34,27 +34,7 @@
 
 unsigned PaymentAPIVersion::current()
 {
-    static unsigned current = [] {
-#if ENABLE(APPLE_PAY_AUTOMATIC_RELOAD_LINE_ITEM) || ENABLE(APPLE_PAY_RECURRING_PAYMENTS) || ENABLE(APPLE_PAY_AUTOMATIC_RELOAD_PAYMENTS) || ENABLE(APPLE_PAY_MULTI_MERCHANT_PAYMENTS) || ENABLE(APPLE_PAY_PAYMENT_ORDER_DETAILS)
-        return 14;
-#elif ENABLE(APPLE_PAY_SELECTED_SHIPPING_METHOD) || ENABLE(APPLE_PAY_AMS_UI)
-        return 13;
-#elif ENABLE(APPLE_PAY_COUPON_CODE) || ENABLE(APPLE_PAY_SHIPPING_CONTACT_EDITING_MODE) || ENABLE(APPLE_PAY_RECURRING_LINE_ITEM) || ENABLE(APPLE_PAY_DEFERRED_LINE_ITEM) || ENABLE(APPLE_PAY_SHIPPING_METHOD_DATE_COMPONENTS_RANGE)
-        return 12;
-#elif ENABLE(APPLE_PAY_SESSION_V11)
-        return 11;
-#elif HAVE(PASSKIT_NEW_BUTTON_TYPES)
-        return 10;
-#elif HAVE(PASSKIT_INSTALLMENTS)
-        if (PAL::getPKPaymentInstallmentConfigurationClass()) {
-            if (PAL::getPKPaymentInstallmentItemClass())
-                return 9;
-            return 8;
-        }
-#endif
-        return 7;
-    }();
-    return current;
```

Comment:
Is is safe to remove these values? Aren't some of these features dependent on OS version?

Reply:
yes because `features` is not guarded and therefore will be available in all future builds regardless of OS version

also, we only throw an error if the given `version` is greater than this value, so the only way this would be problematic (i.e. a compat problem) is if some page out on the web was already using `version: 15` as that would no longer throw an error

Reply:
(we talked about this offline, and need to preserve these values since we ship latest WebKit to older versions of macOS)